requirements/common.txt

```diff
@@ -2,6 +2,7 @@ future
 psutil
 wheel
 requests
+simplejson
```

Comment:
let's just double check that other requirements.txt also need this or not and whether json is used anywhere else in the codebase

Reply:
i wasn't sure where to put this, so i chose common.txt. it's unlikely simpljson is used elsewhere. if that's the case - where do you propose to move this?